machina/traj/epi_functional.py

```diff
@@ -207,6 +207,16 @@ def compute_pseudo_rews(data, rew_giver, state_only=False):
     return data
 
 
+def compute_diayn_rews(data, discrim):
+    epis = data.current_epis
+    for epi in epis:
+        obs = torch.tensor(epi['obs'], dtype=torch.float, device=get_device())
+        with torch.no_grad():
+            rews, info = discrim(obs)
```

Comment:
Is this `discrim` wrapped by calc_rewards?

Reply:
done by wrapped at run_diayn,py
https://github.com/DeepX-inc/machina/pull/170/commits/cd539a0367cac47592185fe9ecdf88a5703e394f